src/Buttons/PayPalButton/PayPalButtonHandler.php

```diff
@@ -31,8 +31,10 @@ public function __construct(PayPalAjaxRequests $ajaxRequests, DataToPayPal $data
     public function bootstrap($enabledInProduct, $enabledInCart)
     {
         if($enabledInProduct){
+            $renderPlaceholder = apply_filters('mollie_wc_gateway_paypal_render_placeholder_product', 'woocommerce_before_add_to_cart_quantity');
```

Comment:
Maybe it should have some kind of indication in the name that it is the hook name? Such as `_hook` prefix instead of `placeholder`.